datamodel/src/main/java/com/hartwig/serve/datamodel/ServeDatabase.java

```diff
@@ -0,0 +1,24 @@
+package com.hartwig.serve.datamodel;
+
+import java.util.Map;
+
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+import com.fasterxml.jackson.databind.annotation.JsonSerialize;
+
+import org.immutables.value.Value;
+import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.Nullable;
+
+@Value.Immutable
+@Value.Style(passAnnotations = { NotNull.class, Nullable.class },
+             jdkOnly = true)
+@JsonSerialize(as = ImmutableServeDatabase.class)
+@JsonDeserialize(as = ImmutableServeDatabase.class)
+public abstract class ServeDatabase {
+
+    @NotNull
+    public abstract String serveVersion();
+
+    @NotNull
+    public abstract Map<RefGenome, ServeRecord> serveDataPerRefGenome();
```

Comment:
Would rename to `records`, the "serve" part is implicit. "recordsPerRefGenome" could also work but I think that is also considered implicit, and I like concise names at the top of the datamodel but that's personal